test/plugins/tokens.test.js

```diff
@@ -185,6 +189,8 @@ describe('token plugin test', () => {
                     scope: ['user']
                 }
             };
+
+            userMock.tokens = tokensGetterMock([tokenMock]);
```

Comment:
Not needed. This is redundant of https://github.com/screwdriver-cd/screwdriver/pull/573/files#diff-ee5dae69c9de402833a459d19c780d18R67